pre-docs/plan_functions.md

```diff
@@ -345,6 +345,10 @@ get_resources('target1,target2', [Package, File[/etc/puppetlabs]])
 
 Parses common ways of referring to targets and returns an array of Targets.
 
+**NOTE:** When using get_targets inside an apply block with a v2
```

Comment:
The text above uses `version 2`, and this text uses `v2`. Can we standardize on one approach? Not sure which is preferred.